test/integration/builder/additional.js

```diff
@@ -190,6 +190,23 @@ module.exports = function(knex) {
             ]);
           });
       });
+
+      // It was needed auto increment ID
+      // it('should return the all columns when single properties as * are given to returning', () => {
```

Comment:
should this be commented out?

Reply:
This feature will display all columns and values. In case, unit test required exactly response data include auto increasement ID. I can't guess this ID so I commented my case.